manifests/charts/gateways/istio-ingress/templates/deployment.yaml

```diff
@@ -266,8 +273,18 @@ spec:
             readOnly: true
           {{- end }}
           {{- end }}
-{{- if $gateway.additionalContainers }}
-{{ toYaml $gateway.additionalContainers | indent 8 }}
+{{- $noSGX := true }}
+{{- range $container := $gateway.additionalContainers }}
+{{ if and ($noSGX) (contains "sds" $container.name) }}
```

Comment:
Hmm, is the logic wrong way around? I mean, should it be (and (not noSGX) ....) to enable SGX?

Reply:
Hi @ipuustin, in fact I may need to change the variable name as `enableSGX`